frontend/src/metabase/containers/UndoListing.jsx

```diff
@@ -51,19 +52,34 @@ UndoToast.propTypes = {
   onDismiss: PropTypes.func.isRequired,
 };
 
+const slideIn = {
+  in: { opacity: 1, transform: "translateY(0)" },
+  out: { opacity: 0, transform: "translateY(100px)" },
+  common: { transformOrigin: "top" },
+  transitionProperty: "transform, opacity",
+};
+
 function UndoToast({ undo, onUndo, onDismiss }) {
+  const [mounted, setMounted] = useState(false);
+
+  useMount(() => {
+    setMounted(true);
+  });
+
   return (
-    <Motion
-      defaultStyle={{ opacity: 0, translateY: 100 }}
-      style={{ opacity: spring(1), translateY: spring(0) }}
+    <Transition
+      mounted={mounted}
+      transition={slideIn}
+      duration={300}
+      timingFunction="ease"
     >
-      {({ translateY }) => (
+      {styles => (
         <ToastCard
           dark
           data-testid="toast-undo"
-          translateY={isReducedMotionPreferred() ? 0 : translateY}
```

Comment:
We can safely remove this as Mantine will skip the transition of the user has reduced motion preferences: https://github.com/mantinedev/mantine/blob/master/packages/%40mantine/core/src/components/Transition/use-transition.ts#L35